README.md

```diff
@@ -149,12 +163,28 @@ $ brew cask install tuntap
 $ lxsim --cpu-type=vexriscv
 ```
 
+The soft CPU `top.bit` needs to be loaded onto the FPGA. For example the ULX3S can use [ujprog](https://github.com/emard/ulx3s-examples/tree/master/bin) like this:
+
+```
+cd $WORKSPACE/litex-boards/litex_boards/targets/soc_basesoc_ulx3s/gateware
+$WORKSPACE/ulx3s-examples/bin/ujprog.exe top.bit
+```
+
 6. Run a terminal program on the board's serial port at 115200 8-N-1.
 
   You should get the BIOS prompt like the one below.
 
 <p align="center"><img src="https://raw.githubusercontent.com/enjoy-digital/litex/master/doc/bios_screenshot.png"></p>
 
+Or, to load the compiled BIOS.bin to be executed by our new soft CPU:
```

Comment:
This can't be right because the BIOS is the one which receives the thing loaded by the `--serial-boot`.

Reply:
@mithro the `BIOS.bin` is not loaded/reloaded by `litex_term`?

It is a bit confusing for me, as indeed there's a primitive BIOS baked in the SoC, but there's also the externally compiled [SoC software BIOS](https://github.com/enjoy-digital/litex/tree/master/litex/soc/software/bios) file too, no? (the RISC-V compiled `BIOS.bin`)